edward/criticisms/evaluate.py

```diff
@@ -16,16 +17,16 @@
   raise ImportError("{0}. Your TensorFlow version is not supported.".format(e))
 
 
-def evaluate(metrics, data, n_samples=500, output_key=None):
+def evaluate(metrics, data, n_samples=500, output_key=None, seed=None):
```

Comment:
Need to update docstring with this new arg. You can follow the standard arg description of seed from TF docstrings.

Reply:
Done.